task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -418,10 +418,11 @@ spec:
         echo "$BASE_IMAGES" >/shared/base_images_from_dockerfile
       computeResources:
         limits:
-          memory: 4Gi
+          cpu: "4"
+          memory: 8Gi
         requests:
-          cpu: 250m
-          memory: 512Mi
+          cpu: "1"
+          memory: 2Gi
```

Comment:
For memory, limit and request should be the same value. This article does a good explaining why: https://home.robusta.dev/blog/kubernetes-memory-limit

Reply:
I would love to have limits == requests, but I assume that is for far bigger fight (and might not make sense from cost point of view).

Reply:
I also think we should have limits=requests but I would do this in steps. Already this PR is increasing the requests for memory and CPU and this will cause pods to be more spread. I would like to see the effect on the worker pools before doing the other step of making memory request=limits.

Reply:
As I understand it, if those values are different we are shooting ourselves in the foot. CPU and memory are different. My concern is only with memory. From the link I shared:

> CPU is fundamentally different than memory. CPU is a compressible resource and memory is not. In simpler terms, you can give someone spare CPU in one moment when it's free, but that does not obligate you to continue giving them CPU in the next moment when another pod needs it. There is no downside to giving away idle CPU, because it's easy and non-violent to reclaim it.

That does not apply to memory.

Reply:
> I also think we should have limits=requests but I would do this in steps

Sure. We do that for the cases where compute resources is already set and this PR is simply adjusting the values. But I think we should make the values match where we are defining a new compute resource.

Reply:
Kubesaw is setting a limit range in tenant namespace so we currently have memory/cpu requests/limits set for all the containers. This PR is setting new compute resource but effectively, it is  changing the value from what the limitrange is setting, which are cpu/memory requests/limits: 10m/2  CPU and 256Mi/2Gi

Reply:
@lcarva Are you OK with proceeding with this change as first step or you think we should do memory requests=limits as part of this first step?

Reply:
@hugares, apologies for not coming back to this sooner. How about we go ahead with this but have a tracker for a follow up?

Reply:
Perfect. Thank you so much.